File: api/src/main/java/com/ruby/rubia_server/core/enums/CampaignStatus.java
package com.ruby.rubia_server.core.enums;

public enum CampaignStatus {
    DRAFT,      // Rascunho, ainda em configuração
    ACTIVE,     // Em execução, enviando mensagens
    PAUSED,     // Pausada temporariamente
    COMPLETED,  // Concluída (todas as mensagens enviadas ou data final atingida)
    CANCELED    // Cancelada antes de concluir
}

File: api-dotnet/Rubia.Server/Enums/CampaignStatus.cs
using System.ComponentModel;

namespace Rubia.Server.Enums;

public enum CampaignStatus
{
    [Description("Rascunho")]
    Draft,      // Rascunho, ainda em configuração
    
    [Description("Ativo")]
    Active,     // Em execução, enviando mensagens
    
    [Description("Pausado")]
    Paused,     // Pausada temporariamente
    
    [Description("Concluído")]
    Completed,  // Concluída (todas as mensagens enviadas ou data final atingida)
    
    [Description("Cancelado")]
    Canceled    // Cancelada antes de concluir
}